apps/backoffice-v2/src/pages/Statistics/Statistics.page.tsx

```diff
@@ -1,9 +1,17 @@
 import React, { FunctionComponent } from 'react';
+import { UserStats } from '@/pages/Statistics/components/UserStats/UserStats';
+import { PortfolioRiskStats } from '@/pages/Statistics/components/PortfolioRiskStats/PortfolioRiskStats';
+import { WorkflowStats } from '@/pages/Statistics/components/WorkflowStats/WorkflowStats';
 
 export const Statistics: FunctionComponent = () => {
   return (
     <div>
-      <h2>Statistics</h2>
+      <h1 className={'pb-5 text-2xl font-bold'}>Statistics</h1>
+      <div className={'flex flex-col space-y-8'}>
+        <UserStats fullName={'John Doe'} />
```

Comment:
Be consistent with the naming. Is it `Stats` or `Statistics`?

Reply:
Naming matches Figma's copy. Would renaming the parts that don't render be an improvement?